packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js

```diff
@@ -44,10 +44,16 @@ export const useBreadcrumbs = props => {
 
             return (
                 breadcrumbData &&
-                breadcrumbData.sort(sortCrumbs).map(category => ({
-                    text: category.category_name,
-                    path: getPath(category.category_url_path, categoryUrlSuffix)
-                }))
+                breadcrumbData
+                    .map(category => ({
+                        category_level: category.category_level,
```

Comment:
This will work but we will return `category_level` now which isn't necessary for rendering.

Reply:
Meh. Good data for someone to have anyway.

Reply:
Was going to comment here too. It's needed for `sortCrumbs`:

```js
const sortCrumbs = (a, b) => a.category_level > b.category_level;
```

We could `map` it again at the end to take it out but yeah, a similar thing happened in `useMagentoRoute` recently where a developer wanted information that we already had but weren't returning because we weren't using it to render.